SampleApplications/Samples/ReverseHello/README.md

```diff
@@ -0,0 +1,19 @@
+# ReverseHello Prototype Readme #
+## Overview ##
+The ReverseHello Prototype codebase includes 7 elements:
+
```

Comment:
TODO

Reply:
move to a /docs folder

---
In reply to: [432161921](https://github.com/OPCFoundation/UA-.NETStandard/pull/998#discussion_r432161921) [](ancestors = 432161921)